docs/content/querying/aggregations.md

```diff
@@ -76,6 +76,58 @@ Computes the sum of values as 64-bit floating point value. Similar to `longSum`
 { "type" : "longMax", "name" : <output_name>, "fieldName" : <metric_name> }
 ```
 
+### First / Last aggregator
+
+First and Last aggregator cannot be used in ingestion spec, and should only be specified as part of queries.
```

Comment:
can we also document that if the rows are rolled up, the first/last value will be a the first/last value in druid row and not the raw data being ingested.